WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dashboard/todaysstats/TodaysStatsCardBuilder.kt

```diff
@@ -1,21 +1,44 @@
 package org.wordpress.android.ui.mysite.cards.dashboard.todaysstats
 
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.dashboard.CardModel.TodaysStatsCardModel
+import org.wordpress.android.fluxc.store.dashboard.CardsStore.TodaysStatsCardError
+import org.wordpress.android.fluxc.store.dashboard.CardsStore.TodaysStatsCardErrorType
 import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.DashboardCards.DashboardCard.TodaysStatsCard
+import org.wordpress.android.ui.mysite.MySiteCardAndItem.Card.DashboardCards.DashboardCard.TodaysStatsCard.TodaysStatsCardWithData
 import org.wordpress.android.ui.mysite.MySiteCardAndItemBuilderParams.TodaysStatsCardBuilderParams
 import org.wordpress.android.ui.stats.refresh.utils.StatsUtils
+import org.wordpress.android.ui.utils.UiString.UiStringRes
 import org.wordpress.android.ui.utils.UiString.UiStringText
 import javax.inject.Inject
 
 class TodaysStatsCardBuilder @Inject constructor(
     private val statsUtils: StatsUtils
 ) {
     fun build(params: TodaysStatsCardBuilderParams) = params.todaysStatsCard?.let {
-        TodaysStatsCard(
-                views = statToUiString(it.views),
-                visitors = statToUiString(it.visitors),
-                likes = statToUiString(it.likes)
+        val error = it.error
+        if (error != null) {
+            createTodaysStatsCardWithError(error)
+        } else {
+            createTodaysStatsCardWithData(it)
+        }
+    }
+
+    private fun createTodaysStatsCardWithError(error: TodaysStatsCardError) = if (shouldShowError(error)) {
+        TodaysStatsCard.Error(
+                title = UiStringRes(R.string.my_site_todays_stat_card_title)
         )
+    } else {
+        null
```

Comment:
Minor: I think we can remove the curly braces from this else block.

Reply:
I know we're not consistent with the curly braces style around `if` control flow statement in the codebase. I last got this feedback: 

> When you have an if/else, either both are on the same line without brackets or if there are in separate like then they Kotlin style wants brackets on both.

which is why I added curly braces in the else block. However, since we can now have if/else in the same line after refactoring in 7c5f5f5, this review comment doesn't require any action now.